packages/angular_devkit/build_angular/src/webpack/plugins/css-optimizer-plugin.ts

```diff
@@ -129,8 +129,23 @@ export class CssOptimizerPlugin {
     const esBuildSupportedBrowsers = new Set(['safari', 'firefox', 'edge', 'chrome', 'ios']);
 
     for (const browser of supportedBrowsers) {
-      const [browserName, version] = browser.split(' ');
+      let [browserName, version] = browser.split(' ');
+
+      // browserslist uses the name `ios_saf` for iOS Safari whereas esbuild uses `ios`
+      if (browserName === 'ios_saf') {
+        browserName = 'ios';
+        // browserslist also uses ranges for iOS Safari versions but only the lowest is required
+        // to perform minimum supported feature checks. esbuild also expects a single version.
+        [version] = version.split('-');
+      }
+
       if (esBuildSupportedBrowsers.has(browserName)) {
+        if (browserName === 'safari' && version === 'TP') {
```

Comment:
I am okay with it, either way.

Reply:
Thought about that but I think it may be better to fail in other cases since the meaning of any future non-numeric version will most likely need special consideration (like the `TP` value).

Reply:
SGTM